.github/workflows/nightly-stage-test-action.yml

```diff
@@ -80,6 +80,11 @@ jobs:
             echo "STABLE_SAM_USERNAME=\"$STAGE_STABLE_SAM_USERNAME\""
             echo "STABLE_SAM_PASSWORD=\"$STAGE_STABLE_SAM_PASSWORD\""
           } >> .env 2>/dev/null
+          if [[ ${{ github.event_name }} -eq "pull_request" ]]; then
+            echo "BASE_URL=https://stage.foo.redhat.com:1337" >> .env
```

Comment:
you have to unset the PROXY environment variable when running against local FE 🔍